static/app/components/events/breadcrumbs/breadcrumbsDrawer.spec.tsx

```diff
@@ -23,42 +23,50 @@ async function renderBreadcrumbDrawer() {
     }));
   render(<BreadcrumbsDataSection {...MOCK_DATA_SECTION_PROPS} />);
   await userEvent.click(screen.getByRole('button', {name: 'View All Breadcrumbs'}));
-  return within(screen.getByRole('complementary', {name: 'breadcrumb drawer'}));
```

Comment:
This was technically correct, but the linter couldn't discover it because it's not based on types (which would be slower to run).

So instead we need to inline the `within()` calls on each line :(